orchagent/portsorch.cpp

```diff
@@ -1884,6 +1894,25 @@ void PortsOrch::doVlanTask(Consumer &consumer)
                 }
             }
 
+            // Process attributes
+            Port l;
```

Comment:
could you rename the variable `l` to `vlan`?

Reply:
Changed to vl